revolution_pi/revolution_pi.go

```diff
@@ -127,3 +132,20 @@ func (b *revolutionPiBoard) Close(ctx context.Context) error {
 	b.logger.Info("Board closed.")
 	return nil
 }
+
+func (b *revolutionPiBoard) DoCommand(ctx context.Context,
```

Comment:
do command that I used for testing the behavior initially. can remove if ya want

Reply:
i think it's fine